community/server-plugin-test/NOTICE.txt

```diff
@@ -42,92 +37,110 @@ Apache Software License, Version 2.0
   Data Mapper for Jackson
   Jackson
   JAX-RS provider for JSON content type
-  Jetty Server
-  Jetty Utilities
+  Jetty :: Http Utility
+  Jetty :: IO Utility
+  Jetty :: Security
+  Jetty :: Server Core
+  Jetty :: Servlet Handling
+  Jetty :: Utilities
+  Jetty :: Webapp Application Support
+  Jetty :: XML utilities
+  Lucene codecs
+  Lucene Common Analyzers
```

Comment:
How come there are Jetty and Lucene things being changed, when Netty is what should be updated?